performance-tests/__snapshots__/performance.test.js.snap

```diff
@@ -0,0 +1,36 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Tram-One - Performance Tests should render lots of elements quickly 1`] = `
```

Comment:
I'm slightly worried that these numbers will change radically when running on different devices, and make it difficult to understand the numbers over time.

We could just snapshot the difference between 10 v 50, 100 v 500, 1000 v 5000, which is what we are actually expecting on in the spec, but that data would be misleading too, since a performance boost linearly would show the same ratios. We could record what the specs of the machine are, but that doesn't make clear what else is running or potentially impacting performance 🤔